templates/paste.html

```diff
@@ -49,4 +49,28 @@
 
 </form>
 </div>
+
+<div>
+<form class="submit-form" method="post" action="/paste/create.go">
```

Comment:
Can you modify the action URL?

http.HandleFunc("/paste/create", createHandler)